scripts/ci-e2e-capi-management-upgrade.sh

```diff
@@ -35,7 +35,7 @@ source "${REPO_ROOT}/hack/ensure-go.sh"
 # Verify the required Environment Variables are present.
 : "${PACKET_API_KEY:?Environment variable empty or not defined.}"
 : "${PROJECT_ID:?Environment variable empty or not defined.}"
-: "${METRO:?Environment variable empty or not defined.}"
+: "${FACILITY:?Environment variable empty or not defined.}"
```

Comment:
Maybe we should do a `0.7.1` shortly after `0.7` to clean this up?

Reply:
That is intentional. That script needs to use Facility as it deploys an old version of CAPP. I don't think we can remove it until we can stop supporting v1alpha3.

Reply:
We chatted about this out-of-band.  At some point, we will have to run this test in a way that does not use facility, but there are a number of ways we could do that (deploy older templates w/ a newer capp, upgrade capp to capi 1.5 when available, ...) and we won't really know which of those options is best until later this year.